cluster/cluster_test.go

```diff
@@ -87,44 +79,31 @@ func (suite *ClusterSuite) TestMemberAdd() {
 		}
 		memberCfg.etcdConfig.Name = "node2"
 		memberCfg.etcdConfig.Dir = "tmp2"
+		memberCfg.etcdConfig.Logger = "zap"
 		memberCfg.etcdConfig.LPUrls = []url.URL{*LPUrl}
 		memberCfg.etcdConfig.LCUrls = []url.URL{*LCUrl}
-		memberCfg.etcdConfig.APUrls = []url.URL{*APUrl}
-		memberCfg.etcdConfig.ACUrls = []url.URL{*ACUrl}
+		memberCfg.etcdConfig.APUrls = []url.URL{*LPUrl}
+		memberCfg.etcdConfig.ACUrls = []url.URL{*LCUrl}
 		memberCfg.etcdConfig.ClusterState = embed.ClusterStateFlagExisting
 
 		c2, err := Join(ctx, memberCfg)
 		require.NoError(t, err)
 		defer c2.Close()
 
-		members, err := c.MemberList(ctx)
+		members, err := c2.MemberList(ctx)
```

Comment:
This was initially to make sure that the initial cluster got another member, if Join ever changes then we still want to ensure that `c` gets this node added because `c2` here might end up being a different cluster if it bootstraps a new cluster or something if logic changes (and if this makes a new cluster of 1 and not 2, someone might just change the `MemberList` check below to equal 1). We can get both `c` and `c2` if you want but I think we should keep a check for `c`

Reply:
Makes sense! I'll change it back to just `c`